### PR TITLE
Support wasm backend

### PR DESCRIPTION
**Pull request by Honry**
Delivers wasm binaries to dist/ foloder for wasm backend.

### CODE REVIEW

**Comment by Honry** [comment]
@BruceDai, @huningxin, PTAL, thanks!

**Comment by BruceDai** [comment]
Thanks @Honry.

I verified locally on browser http://127.0.0.1:8080/test/?backend=wasm with workload
```
--- a/src/nn/context.ts
+++ b/src/nn/context.ts
@@ -63,7 +63,7 @@ export class MLContext {
   get tf(): unknown {
     // Set directory of wasm binaries for 'wasm' backend
     wasm.setWasmPaths(
-        'https://webmachinelearning.github.io/webnn-polyfill/dist/');
+        'http://127.0.0.1:8080/dist/');
     return tf;
   }
 }
```
Now test result of total 615 tests: passes: 566 failures: 48 pending: 1

The failures tests are

- some conv2d transpose with "AssertionError at checkValue"
- some maxPool2d dilations with "**Error: Kernel 'SpaceToBatchND' not registered for backend 'wasm'**"
- some resample upsample nearest with "**Error: Kernel 'ResizeNearestNeighbor' not registered for backend 'wasm'**"
- some CTS maxPool2d / averagePool2d with "AssertionError at checkValue"

**Comment by Honry** [comment]
Thanks @BruceDai ! Are all the failure tests expected for wasm backend?

**Review by huningxin — COMMENTED** [review]
Thanks @Honry , it is a good feature. Looks good with two nits. Please take a look.

File: src/nn/context.ts [hunk 1]
```diff
@@ -60,6 +61,9 @@ export class MLContext {
   /** @internal */
   // Expose tf.js for backend debugging.
   get tf(): unknown {
+    // Set directory of wasm binaries for 'wasm' backend
+    wasm.setWasmPaths(
+        'https://webmachinelearning.github.io/webnn-polyfill/dist/');
```
Comment (huningxin) [inline src/nn/context.ts:66]: BTW, do you think we can set the path to TensorFlow.js URL
```
https://unpkg.com/@tensorflow/tfjs-backend-wasm@${tf.version_core}/dist/
```

So we can avoid host our own version.

**Comment by Honry** [comment]
@huningxin, thanks for your comments, fixed the nits, PTAL again, thanks!

File: src/nn/context.ts (hunk 1 above)
Comment (Honry) [reply src/nn/context.ts:66]: Good idea! I wasn't aware that there're official wasm binaries could be referenced. I will update it soon.

**Comment by Honry** [comment]
@huningxin, I've updated the PR to reuse the wasm binaries from TensorFlow.js, PTAL, thanks!

**Review by huningxin — APPROVED** [review]

**Pull request merged**